pom.xml

```diff
@@ -153,35 +153,35 @@
         <maven-spotbugs-maven-plugin.version>3.1.3</maven-spotbugs-maven-plugin.version>
 
         <!-- Maven plugin dependency versions -->
-        <maven-plexus-compiler-javac-errorprone.version>2.8.2</maven-plexus-compiler-javac-errorprone.version>
-        <maven-error-prone-core.version>2.1.1</maven-error-prone-core.version>
+        <maven-plexus-compiler-javac-errorprone.version>2.8.5</maven-plexus-compiler-javac-errorprone.version>
+        <maven-error-prone-core.version>2.3.2</maven-error-prone-core.version>
 
         <!-- Dependency versions -->
         <dependency.http-client-signature.version>4.0.9</dependency.http-client-signature.version>
-        <dependency.apache-http-client.version>4.5.6</dependency.apache-http-client.version>
+        <dependency.apache-http-client.version>4.5.7</dependency.apache-http-client.version>
         <dependency.bouncycastle.version>1.60</dependency.bouncycastle.version>
```

Comment:
That must have come out since I did my changes. We should.

Reply:
Can you do that upgrade in the http-signatures library so that its BC version is at the same number? I'll update this library so that it uses the 1.61 version explicitly.